api/src/controllers/AuthenticationController.ts

```diff
@@ -157,23 +158,23 @@ export default class AuthenticationContoller extends ErrorController {
         throw ErrorController.BadRequestError();
       }
       await redis.del(user.sub);
-      const userDetails: UserStoredAttributes | null = await this.userService.getUser(user.sub);
+      const userDetails: UserStoredAttributes | null = await this.userService.getUserByUuid(user.sub);
```

Comment:
Just a suggestion, maybe `getUserByUniqueId` as it explains it a bit better.

Reply:
That is more likely to be mistaken for matching on `id` rather than `uuid`, IMO.

Reply:
Happy then.